config.js.example

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Ideally the actual config.js will be added to .gitignore and we'll only push up the .example file, but Travis isn't playing nicely without this file committed to the repo. Will sort that bit out in a later PR.